modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1446,6 +1446,77 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Reuse my object's existing fields when creating a new form view"
+	@priority = "5"
+	test ReuseExistingFormViewFields {
+		property portal.acceptance = "false";
```

Comment:
Hi @nkkwon , could you please remove this annotation related to the acceptance test?